nvflare/edge/emulator/emulator_config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "endpoint": "http://localhost:4321",
+  "num_devices": 16,
+  "processor": {
+    "path": "sample_task_processor.SampleTaskProcessor",
+    "args": {
+      "data_file": "data/data.csv",
```

Comment:
We need to somehow allow different devices to use different data file.

Reply:
Added variable substitution to support this.